apps/parsley/src/utils/string/index.ts

```diff
@@ -135,7 +135,14 @@ export const trimSeverity = (line: string) => {
  * @returns true if line is the failing log line, false otherwise
  */
 export const isFailingLine = (line: string, failingCommand: string) => {
+  console.log(failingCommand);
   const failedExpression = `${failingCommand} failed`;
   const timeoutExpression = `${failingCommand} stopped early`;
-  return line.includes(failedExpression) || line.includes(timeoutExpression);
+  // The "Finished" command is matched when the task status is set to fail by the user.
+  const finished = `Finished command ${failingCommand}`;
```

Comment:
nit. could we call it `finishedExpression` (or something similar) to match the other variables? `finished` sounds like it might be a boolean